proposals/intptr-operators.md

```diff
@@ -0,0 +1,54 @@
+# Operators should be exposed for `System.IntPtr` and `System.UIntPtr`
+
+* [x] Proposed
+* [ ] Prototype: [Complete](https://github.com/PROTOTYPE_OWNER/roslyn/BRANCH_NAME)
+* [ ] Implementation: [In Progress](https://github.com/dotnet/roslyn/BRANCH_NAME)
+* [ ] Specification: [Not Started](pr/1)
```

Comment:
Where should these links point to while there is no prototype, implementation or spec? I don't think dead links are a good option.

Also, this probably shouldn't say that prototype is complete or that implementation is in progress, since it's not true, AFAIK.